MNNparcellation/bin/MNNparcellation

```diff
@@ -29,119 +29,140 @@ from MNNparcellation.inc.CSParcellation import Parcellation as CSP
-		del vertices_plot, faces_plot, normal_plot
-		Parcel.PrepareData(coordinate, Connectivity, Excluded_seeds)
-		Parcel.Parcellation_agg(coordinate, Connectivity, Excluded_seeds,
- 					Regions, SM, Mesh_plot, cvth)
-		# run the parcellation algorithm
+    parser = argparse.ArgumentParser(description=("Parcellation based on MNN"
+                                                  "condition."))
+    parser._action_groups.pop()
+    required = parser.add_argument_group('required arguments')
+    optional = parser.add_argument_group('optional arguments')
+    required.add_argument('-i', action="store", dest='input', type=str,
+                          help='''path to input file .mat that contain cortical
+                          mesh details (Faces, Normal, Vertices, VertConn)''')
+    required.add_argument('-o', action="store", dest='save', type=str,
+                          help='''path to output folder where to save
+                          the results''')
+    required.add_argument('-t', action="store", dest='tractograms', type=str,
+                         help='''folder that contains the tractograms"
+                         format ".nii.gz"''')
+    required.add_argument('-tb', action="store", dest='tract_name', type=str,
+                          help='''prefix of the tractogram name
+                          (tract_name_x_y_z.nii.gz)''')
+    required.add_argument('-seed', action="store", dest='coordinates',
+                          type=str, help='''path of coordinates of seeds in the
+                          diffusion space''')
+    required.add_argument('-sm',  dest='SM', default=['Cosine'],
+                          help='''Similarity measure: Cosine (default),
+                          "Tanimoto, Ruzicka, Motyka''')
+
+    required.add_argument('-n',  dest='NR', default=[100], help='''number T used
+                          to stop merging big regions defaul''')
+
+    optional.add_argument('-cvth', action="store", dest='cv', type=float,
+                          default=np.Inf, help='''variation coefficient is
+                          used to stop merging regions with low homogeneity''')
+
+    optional.add_argument('-nodif', action="store", dest='nodif',
+                          help='''path to white matter mask used to reduce
+                          memory usage''')
+    optional.add_argument('-v', action="store", dest='verbose', type=int,
+                          default=0, help='''Enable (1) showing results in
+                          terminal''')
+
+    optional.add_argument('-m', action="store", dest='merge', type=int,
+                          default=0, help='Merge option: 0 (default)')
+
+    optional.add_argument('-Ex', action="store", dest='excluded',
+                          help='''file that contain seeds to be excluded from
+                          the parcellation''')
+    Arg = parser.parse_args()  # read different parameters
+    run_parcellation(Arg)
```

Comment:
# recomendation
This is ok, however, I'll recomend to change the signature of "run parcellation" in a way such you can call it as "run_parcellation(args.input, args.save, args.merge, args.excluded... etc)". This is exactly what I proposed in my python-talk. Check this script: https://github.com/AthenaEPI/logpar/blob/master/bin/cifti_parcellate